core/src/components/toast/test/a11y/toast.e2e.ts

```diff
@@ -230,3 +230,48 @@ configs({ directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
     });
   });
 });
+
+/**
+ * High contrast mode tests
+ */
+configs({ directions: ['ltr'], themes: ['high-contrast-dark', 'high-contrast-light'] }).forEach(
+  ({ title, config, screenshot }) => {
+    test.describe(title('toast: high contrast: buttons'), () => {
+      test.beforeEach(async ({ page }) => {
+        await page.setContent(
+          `
+        <ion-toast is-open="true" header="Testing" message="Hello world"></ion-toast>
+        <script>
+          const toast = document.querySelector('ion-toast');
+          toast.buttons = [
+            { text: 'Cancel', role: 'cancel' },
+            { text: 'OK' }
+          ];
+        </script>
+      `,
+          config
+        );
+      });
+
+      test('should not have visual regressions', async ({ page }) => {
```

Comment:
Do we need screenshot tests for this? I don't think we added any per-component screenshots for dark mode, since the color updates were handled in dedicated theme tests. (We should definitely keep the Axe test though.)

Reply:
`ion-toast` has custom styles for the different high contrast themes that are not covered by the theme tests.

Reply:
Ah, true! Fair enough 👍